docs/stargz-estargz.md

```diff
@@ -1,81 +1,227 @@
+
+- **`mode`** *int64*
+
+  This OPTIONAL property contains the permission and mode bits.
+
+- **`uid`** *uint*
+
+  This OPTIONAL property contains the user ID of the owner of this file.
+
+- **`gid`** *uint*
+
+  This OPTIONAL property contains the group ID of the owner of this file.
+
+- **`userName`** *string*
+
+  This OPTIONAL property contains the username of the owner.
+
+- **`groupName`** *string*
+
+  This OPTIONAL property contains the groupname of the owner.
+
+- **`offset`** *int64*
+
+  This OPTIONAL property contains the offset of the gzip header of the regular file or chunk in the archive.
+
+- **`devMajor`** *int*
+
+  This OPTIONAL property contains the major device number for character and block device files.
+
+- **`devMinor`** *int*
+
+  This OPTIONAL property contains the minor device number for character and block device files.
+
+- **`xattrs`** *string-bytes map*
+
+  This OPTIONAL property contains the extended attribute for the tar entry.
+
+- **`digest`** *string*
+
+  This OPTIONAL property contains the OCI [Digest](https://github.com/opencontainers/image-spec/blob/v1.0.1/descriptor.md#digests) of the regular file contents.
+  TOCEntries of non-empty `reg` file MUST set this property.
+
+- **`chunkOffset`** *int64*
+
+  This OPTIONAL property contains the offset of this chunk in the regular file payload.
+  Note that this is the offset of this chunk in the decompressed file content.
+  TOCEntries of `chunk` type MUST set this property.
+
+- **`chunkSize`** *int64*
+
+  This OPTIONAL property contains the decompressed size of this chunk.
+  The last `chunk` in a `reg` file or `reg` file that isn't chunked MUST set this property to zero.
+  Other `reg` and `chunk` MUST set this property.
+
+- **`chunkDigest`** *string*
+
+  This OPTIONAL property contains an OCI [Digest](https://github.com/opencontainers/image-spec/blob/v1.0.1/descriptor.md#digests) of this chunk.
+  TOCEntries of non-empty `reg` and `chunk` MUST set this property.
+  This MAY be used for verifying the data of this entry in the way described in [Content Verification in eStargz](/docs/verification.md).
+
+### Footer
+
+At the end of the archive, a *footer* MUST be appended.
+This MUST be an empty gzip member ([RFC1952](https://tools.ietf.org/html/rfc1952)) whose [Extra field](https://tools.ietf.org/html/rfc1952#section-2.3.1.1) contains the offset of TOC in the archive.
+The footer MUST be the following 51 bytes (1 byte = 8 bits in gzip).
+
+```
+- 10 bytes  gzip header
+- 2  bytes  XLEN (length of Extra field) = 26 (4 bytes header + 16 hex digits + len("STARGZ"))
+- 2  bytes  Extra: SI1 = 'S', SI2 = 'G'
+- 2  bytes  Extra: LEN = 22 (16 hex digits + len("STARGZ"))
+- 22 bytes  Extra: subfield = fmt.Sprintf("%016xSTARGZ", offsetOfTOC)
+- 5  bytes  flate header
+- 8  bytes  gzip footer
+(End of eStargz)
+```
+
+*NOTE that though eStarz is compatible to tar.gz, it's not compatible to stargz by Google CRFS.*
```

Comment:
Needs full explanation. The footer differs (for RFC compliance) but TOC is compatible, right?

Reply:
Yes. Added `Notes on compatibility with stargz` subsection for explaining this.